src/tests/test_api.py

```diff
@@ -360,3 +360,34 @@ def test_broken_glencoe_response(self):
         self.assertEqual(resp.status_code, 400) # bad request
         self.assertTrue(utils.partial_match(expected_resp, resp.json))
         self.assertTrue(resp.json['trace'].startswith('Traceback (most'))
+
+    def test_upload_xml_no_iiif_deposit(self):
+        "api returns valid article-json with valid values for image widths and heights when iiif returns a 404"
+        xml_fname = 'elife-24271-v1.xml'
+        xml_fixture = join(self.fixtures_dir, xml_fname)
+
+        expected_lax_resp = {
+            'status': conf.VALIDATED,
+            'override': {},
+            'ajson': base.load_ajson(xml_fixture + '.json')['article'],
+            'message': None # this should trigger an error when logged naively by api.py but doesn't...
+        }
+
+        # don't call lax
```

Comment:
if it doesn't call lax, why we need to patch it?

Reply:
because it will try to talk to lax if we don't. I thought it was clear, but what I'm testing here is what happens when `iiif.iiif_info` returns an empty result `{}` from the api. the response should be `200`, the response body should successfully validate as article json.